R/estimate_infections.R

```diff
@@ -42,8 +42,10 @@
 #'  additional compute. Must also contain the `boundary_scale` (multiplied by half the range of the input time series). Increasing this 
 #' increases the accuracy of the approximation at the cost of additional compute. 
 #' See here: https://arxiv.org/abs/2004.11408 for more information on setting these parameters.
-#' Can optionally also contain the  `lengthscale_mean` and `lengthscale_sd`. If these are specified this will override 
-#' the defaults of 0 and 2 (normal distributed truncated at zero).
+#' Must also contain the  `lengthscale_alpha` and `lengthscale_beta`. These tune the prior of the lengthscale. Principled 
+#' values can be obtained using `tune_inv_gamma` which optimises based on the desired truncation (which should be based on the scale
+#' of the observed data). The default is tuned to have 98% of the density of the distribution between 2 and 21 days. Finally the list must 
```

Comment:
By scale of the observed data, do you mean something along the lines of the length and frequency of the time series?

Reply:
Yes exactly. If the lengthscale is > the length of the timeseries then moving around it doesn't change the fit so the MCMC chain will get stuck. For lengthscales smaller than the smallest observed increment your essentially interpolating (though how this interacts with a latent process I don't know).